private/rules/javadoc.bzl

```diff
@@ -127,6 +141,12 @@ javadoc = rule(
             This information is only used by javadoc targets depending on this target.
             """,
         ),
+        "doc_resources": attr.label_list(
+            doc = "Resources to include in the javadoc jar.",
```

Comment:
are the resources placed top level or are their folder structure kept?
I noticed for data as resources it differs or is controllable.

Reply:
I just put everything at the top level since that's what our needs were.